scripts/_helpers.py

```diff
@@ -769,3 +773,29 @@ def filter_codes(c_list, iso_coding=True):
     full_codes_list = filter_codes(list(set(full_codes_list)), iso_coding=iso_coding)
 
     return full_codes_list
+
+
+# Function to get the last Git commit message
+def get_last_commit_message():
```

Comment:
Please, remove the comment "# Function to ..." and add a docstring instead:
"""
Function to get ...
"""

You can use as a template other functions